packages/duplicate-package-checker-webpack-plugin/package.json

```diff
@@ -50,8 +50,8 @@
   "devDependencies": {
     "@babel/cli": "^7.22.10",
     "@babel/core": "^7.22.10",
-    "@cerner/eslint-config-terra": "^5.5.1",
-    "@cerner/jest-config-terra": "^1.6.2",
+    "@cerner/eslint-config-terra": "5",
+    "@cerner/jest-config-terra": "1",
```

Comment:
These syntax are identical according to the [npm docs](https://docs.npmjs.com/about-semantic-versioning#using-semantic-versioning-to-specify-update-types-your-package-can-accept). The reason this was changed is so that the numbers don't constantly get bumped when we make changes to these packages. Since these are devDependencies, these are not relevant for consumers. This change was made so that we don't have unnecessary releases simply because there are devDependency updates, such as in #823.